cms/envs/test.py

```diff
@@ -264,3 +264,7 @@
 
 # Dummy secret key for dev/test
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+# This is practically unused but needed by the oauth2_provider package, which
+# some tests in common/ rely on.
+OAUTH_OIDC_ISSUER = 'https://example.com/oauth2'
```

Comment:
Instead of re-defining this here, you should add `OAUTH_OIDC_ISSUER` to the list of settings imported from lms.envs.test.  That is, add it to L29 in this file.